config/speechd.conf

```diff
@@ -197,16 +197,22 @@ SymbolsPreprocFile "symbols-fallback.dic"
 #       "pulse" - PulseAudio
 #       "alsa"  - Advanced Linux Sound System
 #       "oss"   - Open Sound System
+#        "pipewire" -   the new low-latency sound server, for near realtime speech performance
 #       "nas"   - Network Audio System
 #       "libao" - A cross platform audio library
-# Pulse audio is the default and recommended sound server. OSS and ALSA
+# Pipe Wire is the default sound server in this configuration,
+# and it is recommended because of the latency related advantages it has over
+# pulseaudio and others,
+#however enabling it should be done with precaution, because it is relatively new and therefore isn't very well tested to work in all configurations supported by pipewire.
+# OSS, Pulse Audio and ALSA
 # are only provided for compatibility with architectures that do not
-# include Pulse Audio. NAS provides network transparency, but is not
-# very well tested. libao is a cross platform library with plugins for
+# include Pipe Wire. NAS provides network transparency, but is not
+# very well tested.
+# libao is a cross platform library with plugins for
 # different sound systems and provides alternative output for Pulse Audio
 # and ALSA as well as for other backends.
 
-# AudioOutputMethod "pulse"
+AudioOutputMethod "pipewire"
```

Comment:
I don't think we want to make it the default yet. Better have it tested by beta-testers before enabling it for everybody.

Reply:
done, I also added some stuff about how the transition should be done